src/components/Nav/components/NavLinks/index.jsx

```diff
@@ -124,18 +135,18 @@ const GordonNavLinks = ({ onLinkClick }) => {
     />
   );
 
-  const linksButton = (
-    <GordonNavButton
-      unavailable={isOnline ? null : 'offline'}
-      openUnavailableDialog={setDialog}
-      divider={false}
-      onLinkClick={() => {
-        onLinkClick();
-        setAreLinksOpen(true);
-      }}
-      linkName={'Links'}
-    />
-  );
+  // const linksButton = (
```

Comment:
I would remove this comment from the code. It's in the git history, so it can always be referenced relatively easily.

Reply:
In fact, you can remove all of the QuickLinks code, including the component, the `areLinksOpen` state, etc.

You should also move the images used on the links page to within the `views/Links` folder.